src/PaymentCalculator.java

```diff
@@ -0,0 +1,11 @@
+public class PaymentCalculator {
+
+    public double nettoYearPayment(Employee employee) {
+        return employee.salary * 12;
+    }
+
+    public double bruttoYearPayment(Employee employee) {
+        double zus = 1000;
```

Comment:
This should be final field of this class.

Reply:
Same with 0.2 which is tax. They don't change ever, so let's make then a final fields.

Reply:
Done.